src/OnboardingSPA/components/SocialMediaForm/urlValidator.js

```diff
@@ -0,0 +1,166 @@
+const SocialMediaSites = {
+	FACEBOOK: 'facebook',
+	TWITTER: 'twitter',
+	INSTAGRAM: 'instagram',
+	YOUTUBE: 'youtube',
+	LINKEDIN: 'linkedin',
+	YELP: 'yelp',
+	TIKTOK: 'tiktok',
+};
+
+// Don't do standard validation for them
+const validationExemption = new Set( [ 'twitter', 'instagram', 'youtube' ] );
+
+// Local copies to prevent unnecessary passing as params
+let errorsDup;
+let setErrorsDup;
+let errorTypesDup;
+let setErrorTypesDup;
+
+// Types of errors
+export const ERROR_TYPES = {
+	NONE: 'none',
+	AD_LINK_ERROR: 'ad-link-error',
+	INVALID_LINK_ERROR: 'invalid-link-error',
+};
```

Comment:
Can we stick to numbers or text either one? I think I saw 400 being set somewhere